packages/blitz/src/render.rs

```diff
@@ -788,6 +788,58 @@ impl ElementCx<'_> {
                                 }
                             }),
                         );
+
+                    if let Some(underline) = &style.underline {
+                        let mut draw_underline = |x: f32, y: f32, w: f32| {
+                            let thickness = 1.0 / 18.0;
+                            let offset = 1.0 / 9.0;
+                            let path = Rect::new(
+                                x as f64,
+                                y as f64 + font_size as f64 * offset,
+                                (x + w) as f64,
+                                y as f64 + font_size as f64 * (thickness + offset),
+                            );
+
+                            scene.fill(
```

Comment:
Consider using `peniko::Line` with `scene.stroke` rather than `kurbo::Rect` with `scene.fill`. It simplifies this code and I think a stroke is more appropriate than a fill for drawing a simple line like this. See https://github.com/linebender/xilem/blob/main/masonry/src/text_helpers.rs#L139